src/services/gatewayClient.ts

```diff
@@ -0,0 +1,66 @@
+import axios from "axios"
+import { Settings } from "../redux/settings/settings.types"
+import { ConnectionCallback } from "../redux/global/global.types"
+import logger from "../logger"
+import { TIMEOUT } from "./config"
+
+export let gatewayInstance = axios.create()
+//todo refactor messages into variables and share them with tests
+
+const gatewayClient = {
+  connect: async (settings: Settings, callback: ConnectionCallback) => {
+    logger.log("api - connect - settings", settings)
+    const baseURL = `${settings.gatewayHost}:${settings.gatewayPort}`
+
+    if (gatewayInstance) {
+      gatewayInstance.defaults.baseURL = baseURL
+      gatewayInstance.defaults.timeout = TIMEOUT
+    } else gatewayInstance = axios.create({ baseURL, timeout: TIMEOUT })
+
+    try {
+      const result = await gatewayInstance.get("/status")
+      if (result.status === 200) {
+        logger.log("api - connect successfully connected to gateway")
```

Comment:
Eventually we might want a consistent style for all logging

Reply:
made that into an issue